pkg/loadbalancer/proxy_test.go

```diff
@@ -181,53 +197,17 @@ func Test_proxyConfig(t *testing.T) {
 			data: &proxyConfigData{
 				HealthCheckPort: 32764,
 				ServicePorts: map[string]data{
-					"IPv4_80": data{BindAddress: "*:80", Backends: map[string]string{
-						"kind-worker":  "192.168.8.2:30497",
-						"kind-worker2": "192.168.8.3:30497",
-					}},
-					"IPv4_443": data{BindAddress: "*:443", Backends: map[string]string{
-						"kind-worker":  "192.168.8.2:31497",
-						"kind-worker2": "192.168.8.3:31497",
-					}},
+					"IPv4_80": data{
+						Listener: endpoint{Address: "0.0.0.0", Port: 80},
+						Cluster:  []endpoint{{"192.168.8.2", 30497}, {"192.168.8.3", 30497}},
+					},
+					"IPv4_443": data{
+						Listener: endpoint{Address: "0.0.0.0", Port: 443},
+						Cluster:  []endpoint{{"192.168.8.2", 31497}, {"192.168.8.3", 31497}},
+					},
 				},
 			},
 			wantConfig: `
-global
-log /dev/log local0
-log /dev/log local1 notice
-daemon
```

Comment:
the fact that `wantConfig` is now `""` seems wrong

Reply:
fixed